tests/python/test_genotypes_api.py

```diff
@@ -0,0 +1,206 @@
+import mock
+import nose
+import nose.tools as asserts
+import json
+
+from cycledash import db
```

Comment:
Hmm, not a great story AFAIK. I think your way is to the style guide, and I'll change this to be "external \n, cycledash-specific \n, in-current-test-directory" -- does that seem reasonable?